tests/memex/__init___test.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+
+import memex
+
+
+def test_set_groupfinder(pyramid_config):
```

Comment:
Luckily (because files called `__init___test.py` give me the heebie-jeebies) I don't think this test is really necessary. See my comment above about a `memex/groups.py` and its `includeme`.